calcom/cal.com
# Fixes translation issue of dynamic variables in workflows

## What does this PR do?

Variables with translations that have more than 2 words like nom de l'événement were falsely translated to 
{NOM_DE L'ÉVÉNEMENT} instead of {NOM_DE_L'ÉVÉNEMENT}

With this PR all spaces get replaced with underscores and not just the first one. 

**Environment**: Staging(main branch) / Production

## Type of change


## How should this be tested?

- Create a workflow with a custom template 
- Add variables to the custom template 
- Go to settings and choose French as your language 
- Go back to your workflow and see that the variables are translated correctly 
- Change language back to English 
- Go back to your workflow
- See that variables are in English again

## Checklist

- I haven't added tests that prove my fix is effective or that my feature works
